src/config/constants.js

```diff
@@ -44,7 +44,7 @@ export const TREE_VIEW_MIN_WIDTH = 350;
 export const TWITTER_MESSAGE_MAX_LENGTH = 270;
 export const MAIL_BREAK_LINE = '%0D%0A';
 
-export const LEFT_MENU_WIDTH = 300;
+export const LEFT_MENU_WIDTH = '18%';
```

Comment:
It's okay for me, but what happens on phone now? Did you set a min width? I know it won't be optimal in any case for phone, but it should at least be usable!